TrustKit/Pinning/pinning_utils.m

```diff
@@ -0,0 +1,88 @@
+/*
+
+ pinning_utils.m
+ TrustKit
+
+ Copyright 2023 The TrustKit Project Authors
+ Licensed under the MIT license, see associated LICENSE file for terms.
+ See AUTHORS file for the list of project authors.
+
+ */
+
+#import "pinning_utils.h"
+#include <dlfcn.h>
+#include "TargetConditionals.h"
+
+bool evaluateTrust(SecTrustRef serverTrust, SecTrustResultType *trustResult, NSError **error) {
+    bool isTrusted = false;
+
+    if (@available(iOS 12.0, macOS 14.0, tvOS 12.0, watchOS 5.0, *)) {
+        CFErrorRef errorRef;
+        isTrusted = SecTrustEvaluateWithError(serverTrust, &errorRef);
+        OSStatus status = SecTrustGetTrustResult(serverTrust, trustResult);
+        if (status != errSecSuccess)
+        {
+            isTrusted = false;
+            NSString *errDescription = [NSString stringWithFormat:@"got status %d", status];
+            *error = [[NSError alloc] initWithDomain:@"com.datatheorem.trustkit" code:1 userInfo:@{NSLocalizedDescriptionKey:errDescription}];
```

Comment:
you should make sure `error` is not null

Reply:
Fixed